examples/modular-drivers/README.md

```diff
@@ -142,7 +153,125 @@ just by adding it as an ordinary dependency in the `dependencies`
 section of the project's `Cargo.toml` file, as noted in the
 `Installation` section above.
 
-## Related Reading
+## Module Implementation
```

Comment:
Since you now have a section for each component, you could now eliminate this `ModuleImplementation` section, and link to the source for each module in its designated section.